platforms/hyperledger-fabric/charts/orderernode/values.yaml

```diff
@@ -121,3 +121,14 @@ proxy:
   #This field contains the external URL of the organization
   #Eg. external_url_suffix: org1.blockchaincloudpoc.com
   external_url_suffix:
+
+config:
+  # Set memory limits of pod  
+  pod:  
+    resources:
+      # Provide the limit memory for node
+      # Eg. limits: 512M
+      limits:
```

Comment:
[nitpick] Since helm chart is independent of the mode of deployment, it might help to have default values set in the `values.yaml` file too. Optionally, helm release file can override it with custom values later.

Reply:
fixed